src/components/common/Navbar/navbar.scss

```diff
@@ -50,11 +58,71 @@
       display: inline-flex;
     }
   }
+
+  [data-store-search-input] {
+    display: none;
+    transition: flex 0.3s ease-in;
+
+    @include media("<notebook") {
+      display: inline-flex;
+      flex: 0;
+      order: 0;
+      width: min-content;
+
+      [data-store-input] {
+        width: 0;
+        padding: 0;
+        border-width: 0;
+
+        &:hover, &:focus, &:focus-visible &:hover:focus-visible {
+          border-width: 0;
+          box-shadow: 0 0;
+        }
+      }
+
+      [data-store-icon] {
+        margin-right: 0;
+        line-height: 0;
+
+        svg {
+          width: var(--space-5);
+          height: var(--space-5);
+        }
+      }
+    }
+  }
+
+  &[data-store-search-expanded="true"] {
+    width: 100%;
```

Comment:
```suggestion
    width: calc(100% - var(--space-7));
```

Reply:
Just to avoid this scenario here 👍 

![image](https://user-images.githubusercontent.com/3356699/154768336-d79a47e9-60f0-4dc4-bbcd-a851b8d34796.png)

Reply:
Great! Thanks!